src/server/rpc/procedures/static-map/static-map.js

```diff
@@ -95,20 +96,23 @@ StaticMap.prototype._recordUserMap = function(socket, options) {
         .then(() => trace(`Stored map for ${socket.roleId}: ${JSON.stringify(map)}`));
 };
 
-StaticMap.prototype.getMap = function(latitude, longitude, width, height, zoom) {
-    var response = this.response,
+
+
+StaticMap.prototype._getMap = function(latitude, longitude, width, height, zoom, mapType, self) {
```

Comment:
It would probably be better if you did `var self = this` at the beginning rather than implicitly doing the variable declaration here.

Is there any other reason why you added `self` as a function parameter?